### PR TITLE
changes, edits and styles

### PR DESCRIPTION
**Pull request by HarikaSree22**
changes:-

1. node border color and edge color when empty validation
2. edge color change for DB and group node
3. predefined height and width of sidebar component images
4. removed reactflow tag under mini-mind map
5. group name validation

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -94,11 +94,18 @@ body,
...
+  pointer-events: none;
+  color: #ffffff;
+}
+
 .dndflow .selectall {
   margin-top: 10px;
 }
 
 .react-flow__node.react-flow__node-ResizableNode,
+.react-flow__node.react-flow__node-customnode,
```
Comment (craxkumar) [inline src/App.css:108]: this is the customnode added by @dheerajsai-g  for testing purpose 
i have asked him to remove it,
i give u missed this one to remove
please remove it!

File: src/components/Sidebar.js [hunk 2]
```diff
@@ -475,7 +475,7 @@ const Sidebar = ({
           ) : (
             <></>
           )}
-          {isEmptyUiSubmit === true ? (
+          {/* {isEmptyUiSubmit === true ? (
```
Comment (craxkumar) [inline src/components/Sidebar.js:478]: u can remove the commented code!
good practises

**Review by craxkumar — CHANGES_REQUESTED** [review]
Please resolve the  review comments

File: src/App.css (hunk 1 above)
Comment (HarikaSree22) [reply src/App.css:108]: DONE

File: src/components/Sidebar.js (hunk 2 above)
Comment (HarikaSree22) [reply src/components/Sidebar.js:478]: done

**Review by craxkumar — APPROVED** [review]
Review OK!

**Review by rohithteja-cmi — APPROVED** [review]

**Pull request merged**